kit/src/main/java/com/oracle/javafx/scenebuilder/kit/library/util/JarExplorer.java

```diff
@@ -32,26 +32,19 @@
  */
 package com.oracle.javafx.scenebuilder.kit.library.util;
 
-import java.io.ByteArrayInputStream;
+import com.oracle.javafx.scenebuilder.kit.library.util.JarReportEntry.Status;
+
 import java.io.IOException;
-import java.lang.reflect.Modifier;
-import java.nio.charset.Charset;
 import java.nio.file.Path;
 import java.util.Enumeration;
 import java.util.jar.JarEntry;
 import java.util.jar.JarFile;
 
-import com.oracle.javafx.scenebuilder.kit.editor.EditorPlatform;
-import com.oracle.javafx.scenebuilder.kit.library.util.JarReportEntry.Status;
-
-import javafx.fxml.FXMLLoader;
-import javafx.scene.Node;
-
 /**
  *
  * 
  */
```

Comment:
This would be a good opportunity to add some Javadoc to the classes explaining what these classes do.

Reply:
This will require some time, probably best for a folluw-up.